crates/tx5-connection/src/config.rs

```diff
@@ -32,6 +32,9 @@ pub struct HubConfig {
     /// The signal config to use.
     pub signal_config: Arc<tx5_signal::SignalConfig>,
 
+    /// The timeout for establishing a WebRTC connection to a peer.
+    pub webrtc_connect_timeout: std::time::Duration,
```

Comment:
We can't just have a single timeout config any more. Adding a new configuration lets the WebRTC connection timeout be set separately to the overall timeout.

Reply:
I'm wondering whether it's better to not expose this setting, and instead subtract some fixed value from the connection timeout, with a minimum, to prevent this situation.
If you don't think it is, then I'd document here that it prevents falling back to relay when exceeding the connection timeout.

Reply:
I tend to agree with @jost-s . Otherwise, people may set the `webrtc_connect_timeout` larger than the overall timeout without being aware of the consequences.

Reply:
I just saw the documentation in the `Config` struct below and it's quite elaborate so I'm probably fine leaving the variable in the config. But it might still be documented better on the `HubConfig` as well here.

Reply:
The overall timeout only exists in `tx5`, not in the `tx5-connection` crate. I have added a general comment to this configuration setting to ask that the `ready` and `send` methods on the `Conn` struct are given enough time for WebRTC to time out and for the connection to fall back to the relay and send the message.